### ISSUE TITLE
Ruby 3 - cannot load rexml/document

### ISSUE THREAD

**Issue by MarcSteven** [realm/jazzy#1260]
I installed jazzy via homebrew and then it shows successfully.
the error as below: I use Ruby3.0.

`<internal:/usr/local/lib/ruby/site_ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- rexml/document (LoadError)
	from <internal:/usr/local/lib/ruby/site_ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /Users/a/.rvm/gems/ruby-2.0.0-p648/gems/jazzy-0.13.7/lib/jazzy/sourcekitten.rb:6:in `<top (required)>'
	from <internal:/usr/local/lib/ruby/s`

**Comment by johnfairh** [comment]
I think there's something wrong with your installation, the gem path shouldn't have `ruby-2.0.0` in it?

Jazzy does work on Ruby 3.0.0:
```shell
; ruby --version
ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-darwin20]
; jazzy -h | head -5
Usage: jazzy

Options
    -o, --output FOLDER              Folder to output the HTML docs to
; jazzy --version
jazzy version: 0.13.7
; uname -a
Darwin Johns-MBP 20.5.0 Darwin Kernel Version 20.5.0: Sat May  8 05:10:33 PDT 2021; root:xnu-7195.121.3~9/RELEASE_X86_64 x86_64
```

**Comment by MarcSteven** [comment]
😂I tried to install it and then it showed it is successful.

**Comment by johnfairh** [comment]
I mean that your Ruby installation looks confused: study the info you posted above, it is mixing up the system Ruby 3.0.0 in `/usr/local/lib` with an RVM Ruby 2.0.0 in `~/.rvm`.

**Comment by johnfairh** [comment]
I'm not sure it helps your problem, but it looks like there is a change we need to make for `rexml` in Ruby 3 - https://stackoverflow.com/questions/65479863/rails-6-1-ruby-3-0-0-tests-error-as-they-cannot-load-rexml.

For now you could install `rexml` manually or use Ruby 2.6/7.

**Comment by MarcSteven** [comment]
@johnfairh  I will try and then ensure the issue, but it's  so weird........The same action occured on two machines, but only one has problem.

**Comment by MarcSteven** [comment]
The solution as blow:
`gem install rexml`


and then all is okay..

**Comment by johnfairh** [comment]
Keeping open to remind me to update the gemspec to include rexml as part of Ruby 2.6 requirement.

**Comment by johnfairh** [comment]
Fixed by #1264 in master.
